modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -58,12 +58,12 @@ class Const {
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1600876608730
+    static public final long APP_TIMESTAMP = 1602683619261
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 5426
+    static public final int APP_BUILDNUM = 5437
 
```

Comment:
Changes to `Const` and other automatically modified files should not be committed